tests/robot-tests/tests/files/four.csv

```diff
@@ -0,0 +1,17 @@
+time_period,time_identifier,geographic_level,country_code,country_name,old_la_code,new_la_code,la_name,filter_one,filter_two,filter_group_one,filter_three,filter_four,filter_group_four,ind_one
```

Comment:
Are there some more appropriate names for these new test files? Names like one, two, three and four are super vague and don't really tell us what they're intended for.

Reply:
Discussed this with Nick and have renamed the files